_docs_v5/calendar.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: docs
+title: Calendar
+toc: true
+---
+
+## Introduction
+
+Calendar allows for the simple execution of commands and scripts. This uses the microservice operator in cosmos to run a multi-threaded application to listen for updates to the timelie schedule and execute on about a seconds accuracy.
+
+![Calendar](/img/v5/calendar/calendar.png)
+
+## Types of Events
+
+- Metadata
+
+- Narrative
+
+- Activity
+
+### Metadata
+
+All Metadata is stored into a single sort set in the backend at millisecond resolution so you can now record any external data outside of the target / packet.  
+
+### Narrative
+
+A simple way to record events on the calendar, for example a test window or anything else...
+
+### Activity
+
+Scheduled on a timeline these can run single commands or kick off scripts to run.
+
+### Adding Timelines
+
+Adding a Timeline to Cosmos.
+
+ - Each timeline consists on several threads so be careful of your compute resources you have as you can overwhelm cosmos with lots of these.
+ - Also note can not have over lapping activities on a single calendar.
+
+### Timeline lifecycle
```

Comment:
This whole section is great implementation detail but it's to the level I want in the code. For the webpage I just want to know what it looks like and why. I like info about what is it doing in the background but only with respect to how I interact with it.